jujugui/static/gui/src/test/test_app.js

```diff
@@ -596,45 +596,6 @@ describe('App', function() {
       assert.equal(conn.messages.length, 0);
     });
 
-    it('displays the login view if credentials are not valid', function() {
-      env.connect();
-      var loginStub = sinon.stub(app, '_renderLogin');
-      app.env.login();
-      // Mimic a login failed response assuming login is the first request.
-      conn.msg({'request-id': 1, error: 'bad wolf'});
-      assert.equal(1, conn.messages.length);
-      assertIsLogin(conn.last_message());
-      assert.equal(loginStub.callCount, 2);
-      assert.equal(loginStub.args[0][0], 'bad wolf');
-      assert.equal(loginStub.args[1][0], 'bad wolf');
-    });
-
-    it('login method handler is called after successful login', function(done) {
-      let localApp = {};
-      const oldOnLogin = Y.juju.App.prototype.onLogin;
-      Y.juju.App.prototype.onLogin = evt => {
-        assert.equal(conn.messages.length, 1);
-        assertIsLogin(conn.last_message());
-        assert.strictEqual(evt.detail.err, null);
-        localApp.on('destroy', () => done());
-        localApp.destroy();
-      };
-      this._cleanups.push(() => {
-        Y.juju.App.prototype.onLogin = oldOnLogin;
-      });
-      localApp = new Y.juju.App({
-        baseUrl: 'http://example.com/',
-        controllerAPI: controller,
-        env: env,
-        controllerSocketTemplate: '/api',
-        viewContainer: container,
-        jujuCoreVersion: '2.0.0'
-      });
-      env.connect();
-      localApp.env.userIsAuthenticated = true;
-      localApp.env.login();
-    });
```

Comment:
Yeah because the `onLogin` method is removed and duplicated functionality the tests which tested that functionality are no longer needed.